static/app/components/sidebar/index.tsx

```diff
@@ -119,11 +123,25 @@ function Sidebar() {
   const activePanel = useLegacyStore(SidebarPanelStore);
   const organization = useOrganization({allowNull: true});
   const {shouldAccordionFloat} = useContext(ExpandedContext);
+  const hasNewNav = useMemo(() => {
+    return organization?.features.includes('navigation-sidebar-v2');
+  }, [organization]);
```

Comment:
not really worth memoing since the boolean already works well with Object.is comparisons downstream and includes is very fast
```suggestion
  const hasNewNav = organization?.features.includes('navigation-sidebar-v2');
```